src/main/java/uk/gov/hmcts/reform/hmc/controllers/HearingManagementController.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.hmc.controllers;
+
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.hmc.service.HearingManagementService;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+public class HearingManagementController {
+
+    private final HearingManagementService hearingManagementService;
+
+    public HearingManagementController(HearingManagementService hearingManagementService) {
+        this.hearingManagementService = hearingManagementService;
+    }
+
+    @GetMapping(path = "/hearing/{id}", produces = APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @ApiResponses(value = {
+        @ApiResponse(code = 204, message = "Hearing id is valid"),
+        @ApiResponse(code = 404, message = "Invalid hearing id")
+    })
+    public void getHearing(@PathVariable("id") Long hearingId,
+                           @RequestParam(value = "isValid", required = false) Boolean isValid) {
```

Comment:
```suggestion
                           @RequestParam(value = "isValid", defaultValue = "false") boolean isValid) {
```
I think we should use this instead and treat the query param like a normal one rather than the current requirements - makes life easier 😄 Then we can just pass `isValid` directly into the next method instead too